cmd/mayactl/app/command/volume_create.go

```diff
@@ -64,20 +57,21 @@ func NewCmdVolumeCreate() *cobra.Command {
 
 	cmd.Flags().StringVarP(&options.size, "size", "", options.size,
 		"volume capacity in GB (example: 10G) (default: 5G")
-
 	return cmd
 }
 
-// Run does tasks related to mayaserver.
-func (c *CmdVolumeCreateOptions) Validate(cmd *cobra.Command) error {
-	if c.volName == "" {
+// Validate varifies whether a volume name has been provided or not followed by
```

Comment:
varifies = verifies
has been provided = is provided

Reply:
done.